networkx/algorithms/connectivity/connectivity.py

```diff
@@ -325,8 +325,9 @@ def neighbors(v):
     kwargs = dict(flow_func=flow_func, auxiliary=H, residual=R)
 
     # Pick a node with minimum degree
-    degree = G.degree()
+    degree = dict(G.degree())
     minimum_degree = min(degree.values())
+    # TODO
```

Comment:
I need to find a way to make it work for `degree = G.degree()`, rather than making it a dictionary.